src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -164,12 +164,17 @@ Action onImageSet
 				//We only need to set a background if the drawArea is non-zero
 				if (!drawArea.HasZeroArea())
 				{
-					var imageBrushBackground = background as ImageBrush;
-					if (imageBrushBackground != null)
+					if (background is ImageBrush imageBrushBackground)
 					{
 						var setBackground = DispatchSetImageBrushAsBackground(view, imageBrushBackground, drawArea, onImageSet);
 						disposables.Add(setBackground);
 					}
+					//TODO: AcrylicBrush on Android
+					//else if (background is AcrylicBrush acrylicBrush)
+					//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)